go/cmd/chunk_sink/session-source-handler.go

```diff
@@ -112,18 +144,40 @@ func (h *SessionSourceHandler) streamSessions(ctx context.Context, request *sspb
 	sessions := h.sessionStorage.GetSessions(recorderID)
 	for _, session := range sessions {
 		if session.IsClosed {
+			// Generate presigned URLs for session files (24 hour expiry)
+			expiry := 24 * time.Hour
```

Comment:
I like this, but this should be the same value as the lifetime of the session, no?

Reply:
No, because you get a new link every time you open the web interface.